### ISSUE TITLE
Allow configuring what is included in the accessible buffer

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#179716]
Specifically, whether or not the `prompt` should be included would be useful now that we have the ability to jump up / down to commands.

cc @jooyoungseo

**Comment by meganrogge** [comment]
It wouldn't be possible on Windows to know where the prompt is and it can have important information, so for now, we are blocked.

**Comment by meganrogge** [comment]
I think that users will have gotten used to this by now and also haven't seen such requests, so closing.
